atlasdb-client/src/main/java/com/palantir/atlasdb/transaction/impl/TransactionConstants.java

```diff
@@ -47,6 +47,7 @@ private TransactionConstants() {
     public static final TransactionStatus ABORTED = TransactionStatuses.aborted();
     public static final TransactionStatus IN_PROGRESS = TransactionStatuses.inProgress();
     public static final TransactionStatus UNKNOWN = TransactionStatuses.unknown();
+    public static final byte[] TTS_IN_PROGRESS_MARKER = getTtsInProgressMarker();
```

Comment:
Why not just do `byte[] TTS_IN_PROGRESS_MARKER = new byte[] {1};`?

Reply:
Yep I agree with that but checkstyle does not so much ` warning: [MutablePublicArray] Non-empty arrays are mutable` :(